.github/workflows/build-macOS11-S3.yml

```diff
@@ -76,14 +76,6 @@ jobs:
           python -m pip install --upgrade pip virtualenv
         shell: bash
 
-      - name: 'Install system headers (OSX 10.14 only)'
```

Comment:
does this conditional need to be removed from `unit-test-runs.yml` as well?

Reply:
Updated, thanks.